src/core/meshcop/dataset_manager.hpp

```diff
@@ -366,8 +372,11 @@ class DatasetManager : public InstanceLocator
         kSendSetDelay   = 5000, // Milliseconds
     };
 
-    bool       mCoapPending : 1;
+    bool       mMgmtPending : 1;
     TimerMilli mTimer;
+
+    otDatasetMgmtSetCallback mMgmtSetCallback        = nullptr;
+    void *                   mMgmtSetCallbackContext = nullptr;
```

Comment:
I myself prefer initializing member here but we should follow the convention that members are initialized in the `initialization-list`.

Reply:
Done.